Tools/buildstream/Makefile

```diff
@@ -19,6 +19,7 @@ EXPORT_ARGS ?=
 
 RSYNC_HOST ?= software.igalia.com
 RSYNC_REMOTE_DIR ?= /var/www/software/webkit-sdk-repo
+RSYNC_ARGS ?=
```

Comment:
I'd make '-vz' the default, so one can override it.

Reply:
Oh sorry I didn't see the comments before landing the PR. I'll follow-up, this is a good suggestion indeed.